qt/python/mantidqt/project/projectsaver.py

```diff
@@ -45,27 +46,47 @@ def save_project(self, directory, workspace_to_save=None, plots_to_save=None):
         # Generate plots
         plots_to_save_list = PlotsSaver().save_plots(plots_to_save)
 
+        # Save interfaces
+        if interfaces_to_save is None:
+            interfaces_to_save = []
+        interfaces = []
+        for interface, encoder in interfaces_to_save:
+            # Add to the dictionary encoded data with the key as the first tag in the list on the encoder attributes
+            try:
+                tag = encoder.tags[0]
+                encoded_dict = encoder.encode(interface, directory)
+                encoded_dict["tag"] = tag
+                interfaces.append(encoded_dict)
+            except Exception as e:
+                # Catch any exception and log it
+                if isinstance(e, KeyboardInterrupt):
+                    raise KeyboardInterrupt(e)
```

Comment:
Do don't think you need to create a new exception object. You can just do:
```python
...
  except Exception as e:
    if isinstance(e, KeyboardInterrupt):
      raise
```

to rethrow the exception object.